hphp/runtime/ext/string/ext_string.php

```diff
@@ -568,7 +569,7 @@ function str_pad(string $input,
  * @return string - Returns the repeated string.
  *
  */
-<<__Native>>
+<<__IsFoldable, __Native>>
 function str_repeat(string $input, int $multiplier): string;
```

Comment:
This seems risky - what about str_repeat('x', 2000000000) ? The resulting static string is going to live forever...